dags/sentinel-2_data_transfer.py

```diff
@@ -245,7 +238,20 @@ def start_transfer(stac_item):
             else:
                 copied_files.append(result)
 
-    if len(copied_files) == 17:
```

Comment:
Do you think we should check all the data copied first?

This will copy the STAC file even if only 2 files were copied to the destination.

Reply:
I'm adding that feature now, but I'd like to measure how much time it adds to runtime.  I'll raise a separate PR with profiling info.  Are you happy for me to merge this and process dead letter queue messages?

Reply:
I think I misunderstood this, I was thinking of copying existing keys to destination.  I'll add a check for the copied objects.